File: Patterns in C/Pattern 3.cpp
# include <stdio.h>
int main()
{
	int rows,space;
	printf("Enter No of Rows ");
	scanf("%d",&rows);
	
	for(int i=1, k=0; i<=rows; i++, k=0)
	{
		for(space=1; space<=rows-i; space++)
		{
			printf("  ");
		}
		while(k != 2*i-1)
		{
			printf("* ");
			k++;
		}
		printf("\n");
	} 
	return 0;
}

File: Patterns in Java/Pattern_3.java
import java.util.Scanner;

public class Pattern_3 {
public static void main(String[] args) {
int rows, space;
Scanner sc = new Scanner(System.in);
System.out.print("Enter No of Rows: ");
rows = sc.nextInt();

for (int i=1, k=0; i<=rows; i++, k=0) {
for (space=1; space<=rows-i; space++) {
System.out.print("  ");
}
while (k != 2*i-1) {
System.out.print("* ");
k++;
}
System.out.println();
}
}
}